### DISCUSSION TITLE
Yellow and yellow/orange roads in Cruiser almost impossible to distinguish while riding in sunshine etc. ---can this be changed ?

### DISCUSSION THREAD

**Discussion by RolfGS** [devemux86/cruiser#602] [Q&A] [answered]
Hello,
I am currently riding thru the French Alps and am very happy with my Kurviger/Cruiser/Samsung S10e navigation solution.
But I repeatedly run in the proble of hitting too small roads because I cannot recogize while driving the contrast between "clear yellow" = very small roads, not good to drive and "orange-yellow" = better roads.

Can this contrast be change to make e.g. the "orange/yellow" colour much better distinguishable?

To make my problem statement clearer please look in attached screenshot:
Rd D 990 is "orange-yellow" - or just a littler darker than "yellow" Rd 86c.
Rd is the "good" road and Rd 86c is the "bad" road here--> Rd 86c should be avoided.

NOW: while driving you cannot distinguish both colours, so very often you hit a "bad"/too small road.

Proposed solution : make contrast "orange-yellow" stronger (more orange) or chose a different colour.

Is this possible ?
Cheers, Rolf

![Screenshot_20240721_142642_Cruiser](https://github.com/user-attachments/assets/26d7202c-30f4-4b11-b6e8-8e3cc606cb12)

**Comment by devemux86** [comment]
@RolfGS thanks for the report.

Do you use the default Biker map theme or another theme?

Have you enabled the "High contrast" map color in settings?

---

Next Cruiser will use a new map theme prepared by @MotoUKRider.

@MotoUKRider what colors are there on these types of roads?

![cruiser-mapsforge-preview-3](https://github.com/mapsforge/mapsforge/assets/125696109/61ec1d3e-f056-4297-af73-53a3f341e4fb)

**Reply by Ciutnik** [reply]
With full respect to the effort @MotoUKRider put into creating map themes, but the night one is not very readable for me. Perhaps it depends on the region and how OSM is mapped, but in my area, all roads are black (dark gray), regardless of their category. In addition, the light outlines of dark minor roads on a dark background are, in my opinion, difficult to read. I think they would look better gray, without outlines.

This can be seen even in the example above - only "blue" and "green" are retained. But "red", "orange" and "yellow" are as black as all the other, unclassified ones. Maybe we could color them a little?

**Reply by devemux86** [reply]
> Maybe we could color them a little?

Indeed, we can have the important road categories with colors in the dark theme too.

**Reply by MotoUKRider** [reply]
@Ciutnik Please bare in mind that it's primarily for "navigation" rather than a "browse" theme. In fact, many other programs auto-default to a standard "day" map when coming out of "navigate". The idea is that there are few "colourful" distractions when driving/riding.

The existing "Dark" theme in the App is not much different - so you can investigate yourself and highlight any "problem" areas here, and I'll replicate and try some alternative fills/shades.

What you see here is the current beta (with hillshading) under Mapsforge:

![cruiser-day-night](https://github.com/user-attachments/assets/284e94d6-33d5-42de-96bc-f2a45b00df13)

The subtle shades ensure that safe operating takes place *and* we try to avoid colour-clashes with route lines.

**Reply by Ciutnik** [reply]
@MotoUKRider 
> Please bare in mind that it's primarily for "navigation" rather than a "browse" theme. 

Definitely yes! Especially at night, when we have to focus a lot of attention on the road. The map is supposed to help us, not distract us. I believe that in night mode you can omit pedestrian paths and building outlines, but road categories are important information when looking for the best route, e.g. when navigation fails. 

I also believe that discreet green areas and waters are helpful in orientation and do not interfere. I keep thinking about the night mode, because, as you say, the day mode can be "richer" because it is also used for planning.

**Comment by RolfGS** [comment]
Hallo, ja ich habe "hoher Kontrast" in den Einstellungen der Kartenfarbe.
Ich weiss leider nicht welches Kartendesign ich habe. Wo bitte kann ich das nachschauen?

**Reply by devemux86** [reply]
@RolfGS 

It is in the menu "Map | Map theme".
Can you try without the "High contrast" option?
Can you send an [OpenStreetMap](https://www.openstreetmap.org/) link of the map area in the screenshot?

Es befindet sich im Menü "Karte | Kartenthema".
Können Sie es ohne die Option "Hoher Kontrast" versuchen?
Können Sie einen [OpenStreetMap](https://www.openstreetmap.org/) Link des Kartenbereichs auf dem Screenshot senden?

**Comment by RolfGS** [comment]
![Screenshot_20240721_154134_Cruiser](https://github.com/user-attachments/assets/2754ee5f-fe78-4999-af29-af11237f52f5)

...please...now with contrast "normal" , think the contrast D 990  to D 86c is not much better. I can try this but I am not sure how good the whole map will be in shunshine...

**Reply by MotoUKRider** [reply]
@RolfGS @devemux86 This looks like the original Biker theme - I see the same colour when loading it here. "Secondary" and "Tertiary" roads are indeed very similar!

**Reply by RolfGS** [reply]
Yes, think you got my "problem". In recent days it happened repeatedly that a) I got tertiary roads in my planed route and b) when leaving the planned route I got into another tertiary road by not being able to distinguish between the coulours while riding, so it got repeatedly "very curvy and bumpy". My good firend following me just had the dry comment " looks like we have different understanding of good curvy roads"...

**Reply by devemux86** [reply]
@RolfGS 

If you select "Motorider" map theme, then it should be a clear difference with orange vs yellow color.
Biker|Motorider
-|-
![1](https://github.com/user-attachments/assets/25f010ce-aa7c-4a1e-a9b9-2cc770dcb92c)|![2](https://github.com/user-attachments/assets/46c74548-0e88-4247-9948-33be2b8a3038)

**Reply by WolfgangZ42** [reply]
The same map section with Motorider theme:
![2024-07-21 17 23 26](https://github.com/user-attachments/assets/5dea661b-4367-4f66-a81d-6d402bdf57e2)

**Reply by RolfGS** [reply]
Entschuldige mein Unwissen, aber wie komme ich zu der Einstellung "Motorider-Motiv" ? Was muss ich am Handy machen?
Ich kann es einfach nicht finden.

**Reply by devemux86** [reply]
@RolfGS 

It is in the menu "Map | Map theme".

Es befindet sich im Menü "Karte | Kartenthema".

**Reply by RolfGS** [reply, marked as answer]
![Screenshot_20240721_191547_Cruiser](https://github.com/user-attachments/assets/1337622a-ee84-4abb-b377-3922fa3f0d81)

Hello,
hands down! I found Moto-Rider ! And even better : when I changed to high contrast the 2 roads come out very clearly - see attached example. Its not what I sent before but it is one of the roads I mistakenly took Friday: Instead of staying on D907 I took D 8 --> in the old card look "Biker" both streets looked like the same yello. But now yellow is tertiary, orange is secondary and almost red is primary (not shown on the screenshot)--> solution seems be "Moto-Rider and high contrast! Will try out tomorrow ...thanks for all your quick assitance - hope I will not have to come back on this again.

**Comment by Leeb64** [comment]
[🌍NormalStyle#Elevate_beta.zip](https://github.com/user-attachments/files/16324789/NormalStyle.Elevate_beta.zip)
Test this (elevate!).
